libbeat/docs/shared-ssl-config.asciidoc

```diff
@@ -143,13 +143,20 @@ The default value is `[TLSv1.1, TLSv1.2, TLSv1.3]`.
 [float]
 ==== `verification_mode`
 
-This option controls whether the client verifies server certificates and host
-names. Valid values are `none` and `full`. If `verification_mode` is set
-to `none`, all server host names and certificates are accepted. In this mode,
-TLS-based connections are susceptible to man-in-the-middle attacks. Use this
-option for testing only.
-
-The default is `full`.
+Controls the verification of certificates. Valid values are:
```

Comment:
This is cribbed from the ES docs: https://github.com/elastic/elasticsearch/blob/7.8/docs/reference/settings/common-defs.asciidoc

Reply:
It's also not clear to me where I should update this section of the example config. I see there's multiple `tmpl` files with identical content, do I edit each one?

Reply:
@andrewkroh wondering the same here. Do we collect the SSL settings from one common template, or do we need the templates to include SSL settings?

Reply:
In https://github.com/elastic/beats/pull/20357 I created a template for the SSL options and then included it in each output config. With this change you'll only need to update the one template.